src/components/HomePage/AccountCard/AccountTransactions/AccountTransaction/AccountTransaction.css

```diff
@@ -25,3 +25,21 @@
   line-height: 24px;
   color: #736E7D;
 }
+
+.AccountTransaction .in-transaction-logo {
+  width: 34px;
```

Comment:
If logos will have the same size everything maybe we can have a css class:
```css
.transaction-logo { 
  width: 34px;
  height: 34px;
}
```

And then use it:
```css 
.AccountTransaction .transaction-logo.out {
  background-image: url('../../../../../images/out-transaction.svg'); 
}
```

Reply:
done! thanks for the catch!